package.json

```diff
@@ -30,6 +30,7 @@
     "danger": "danger-ci",
     "lint": "eslint '@(packages|scripts)/**/{*.js,package.json}' --ignore-pattern node_modules --ignore-pattern storybook-dist",
     "postbuild": "rimraf \"./packages/*/dist/{,**/}__*__\"",
+    "preinstall": "node -e 'process.env.CI||process.exit(1)' || npx npm-is npm",
```

Comment:
Makes sense to me. Does `npm-is` not need to be added to the list of dependencies?

Reply:
@zetlen I think you meant `node -e 'process.env.CI||process.exit(1)' || npx npm-is yarn`?

Reply:
@dpatil-magento I did, sorry. I was testing it and forgot to put it back.

@jimbo It does not! That’s why we use npx to run it, because it has to work before any dependencies are installed.